gtk4/src/auto/file_chooser.rs

```diff
@@ -86,7 +86,7 @@ pub trait FileChooserExt: 'static {
     fn set_current_folder<P: IsA<gio::File>>(&self, file: &P) -> Result<(), glib::Error>;
 
     #[doc(alias = "gtk_file_chooser_set_current_name")]
-    fn set_current_name<P: AsRef<std::path::Path>>(&self, name: P);
+    fn set_current_name(&self, name: &str);
```

Comment:
Is this change correct?

Reply:
Yes, it takes a file name, not a file path.